README.md

```diff
@@ -0,0 +1,174 @@
+<a name="readme-top"></a>
+
+  <h3><b>Leaderboard Porject</b></h3>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
```

Comment:
![image](https://github.com/Errytagedesign/gameLeaderboard/assets/93039359/a4b177f0-7411-48d1-bbdd-5ffaa28a8b3c)

- [ ] It would be nice if you fix the Run tests link because right now it is broken 😨 Kindly fix it to make your readme descriptive and professional 👍 as suggested by the previous reviewer 😊